graylog2-web-interface/src/components/inputs/InputSetupWizard/steps/StartInputStep.tsx

```diff
@@ -45,18 +46,25 @@ export type ProcessingSteps =
   | 'createStream'
   | 'startStream'
   | 'createPipeline'
+  | 'connectPipeline'
   | 'setupRouting'
   | 'deleteStream'
   | 'deletePipeline'
   | 'deleteRouting'
   | 'result';
 
 const StartInputStep = () => {
+  const ExtraSetupWizardStep = PluginStore.exports('inputSetupWizard').find(
```

Comment:
We can make this more idiomatic (and avoiding the `PluginStore` global singleton) by doing:
```suggestion
  const inputSetupWizards = usePluginEntities('inputSetupWizard');
  const ExtraSetupWizardStep = useMemo(
    () => inputSetupWizards?.find((plugin) => !!plugin.ExtraSetupWizardStep)?.ExtraSetupWizardStep,
    [],
  );
```